src/app/authentication/login/login.component.ts

```diff
@@ -37,21 +45,41 @@ export class LoginComponent implements OnInit {
       password:['', Validators.required],
     })
   }
-  buildSignUpFrom(){
+  buildSignUpForm(){
     this.signUpForm = this.formBuilder.group({
+      password:[''],
+      phoneNumber: this.formBuilder.array([this.formBuilder.control('')]),
+      hobbies:[['']],
+      duration:[''],
+      startDate:[''],
+      endDate:[''],
+      interests: [['']],
+      url:[''],
+      rememberMe:[false]
 
     })
   }
   submitForm(){
     console.log(this.loginForm.value)
   }
+  submitSignUpForm(){
+    console.log(this.signUpForm.value)
+  }
+
+  addNewNumber(e) {
+    this.phoneNumber.push(this.formBuilder.control(''));
+    // i can't seem to be able to get the value of the phone number inorder to push it to an array
```

Comment:
This is to add an empty form control to the array not to get a value to push (The result of this should give you an added textbox for a new number)